src/objects/zcl_abapgit_object_msag.clas.abap

```diff
@@ -179,18 +186,70 @@ CLASS ZCL_ABAPGIT_OBJECT_MSAG IMPLEMENTATION.
 
 
   METHOD zif_abapgit_object~delete.
+    DATA: lv_t100a          TYPE t100a,
+          lv_frozen         TYPE flag,
+          lv_message_id     TYPE arbgb,
+          lv_trkey1         TYPE trkey,
+          lv_access_granted TYPE abap_bool.
+
+* parameter SUPPRESS_DIALOG doesnt exist in all versions of FM RS_DELETE_MESSAGE_ID
+* replaced with a copy
+    lv_message_id = ms_item-obj_name.
+    IF ms_item-obj_name EQ space.
+      zcx_abapgit_exception=>raise( 'Error from (copy of) RS_DELETE_MESSAGE_ID' )."blank message id
+    ELSE.
```

Comment:
ELSE not needed, nesting can be avoided, as the IF raises an exception

Reply:
Good point, fixed